psycop/common/model_training_v2/hyperparameter_suggester/suggesters/logistic_regression_suggester.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+from typing import Any
+
+import optuna
+
+from psycop.common.model_training_v2.hyperparameter_suggester.suggesters.base_suggester import (
+    Suggester,
+)
+
+FloatSpaceT = tuple[float, float, bool]
+
+
+@dataclass(frozen=True)
+class FloatSpace:
+    low: float
+    high: float
+    logarithmic: bool
+
+    def suggest(self, trial: optuna.Trial, name: str) -> float:
+        return trial.suggest_float(
+            name=name,
+            low=self.low,
+            high=self.high,
+            log=self.logarithmic,
+        )
+
+
+class LogisticRegressionSuggester(Suggester):
+    # XXX: Should we move this to the logistic regression creator, and then have the creator also subclass suggester?
```

Comment:
I agree that it's appealing to have them together, but I think you're right that decorating them will be hard then